service/awsconfig/v4/resource/cloudformation/adapter/instance.go

```diff
@@ -34,17 +39,17 @@ func (i *instanceAdapter) getInstance(cfg Config) error {
 	clusterID := key.ClusterID(cfg.CustomObject)
 	s3URI := fmt.Sprintf("%s-g8s-%s", accountID, clusterID)
 
-	cloudConfigConfig := SmallCloudconfigConfig{
+	c := SmallCloudconfigConfig{
 		MachineType:        prefixMaster,
 		Region:             cfg.CustomObject.Spec.AWS.Region,
 		S3URI:              s3URI,
 		CloudConfigVersion: cloudconfig.MasterCloudConfigVersion,
 	}
-	smallCloudConfig, err := SmallCloudconfig(cloudConfigConfig)
+	rendered, err := templates.Render(key.CloudConfigSmallTemplates(), c)
 	if err != nil {
 		return microerror.Mask(err)
 	}
-	i.MasterSmallCloudConfig = smallCloudConfig
+	i.MasterSmallCloudConfig = base64.StdEncoding.EncodeToString([]byte(rendered))
```

Comment:
The general rendering returns a string. The only difference for the small cloud config rendering was the base64 encoding, which is now done here.